packages/frontend/components/authorize/AuthorizeClaim.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+
+
+
+interface Props {
+  claim: keyof typeof claims,
```

Comment:
In oauth specification, this is called scope, It's smarter to call things properly, so we don't have to look for what other people named a specific component